.eslintrc.js

```diff
@@ -236,14 +236,16 @@ module.exports = {
     'rxjs/no-subject-unsubscribe': 'error',
 
     'etc/throw-error': 'error',
-    'etc/no-deprecated': 'warn',
+    'etc/no-deprecated': 'off',
```

Comment:
Could you share information on errors you're getting for each rule as a comment here?

Reply:
Added on comment https://github.com/sourcegraph/eslint-config/pull/247#discussion_r877756140

Reply:
We should be able to leave it on despite having new warnings since it's not an `error`. What do you think?